blobfuse/utilities.cpp

```diff
@@ -303,8 +303,11 @@ int azs_getattr(const char *path, struct stat *stbuf)
     if (strlen(path) == 1)
     {
         stbuf->st_mode = S_IFDIR | 0770; // TODO: proper access control.
+        stbuf->st_uid = fuse_get_context()->uid;
+        stbuf->st_gid = fuse_get_context()->gid;
         stbuf->st_nlink = 2; // Directories should have a hard-link count of 2 + (# child directories).  We don't have that count, though, so we jsut use 2 for now.  TODO: Evaluate if we could keep this accurate or not.
         stbuf->st_size = 4096;
+        stbuf->st_mtime = time(NULL);
```

Comment:
If we're going to address the timing issue, we should decide what to do about directories.  I'm not convinced that "right now" is a good option for a directory timestamp; that could be confused for a correct value, where as '1 Jan 1970' is more clearly wrong.

Reply:
Reset the directory timestamps back to 1970